lib/structs/responses/common.cpp

```diff
@@ -410,8 +410,8 @@ parse_timeline_events(const json &events,
                     break;
                 }
                 case MsgType::Location: {
-                    /* events::RoomEvent<events::msg::Location> location = e; */
-                    /* container.emplace_back(location); */
+                    events::RoomEvent<events::msg::Location> location = e;
+                    container.emplace_back(location);
```

Comment:
Can you write that in the style of the lines above, i.e.:

```suggestion
                    container.emplace_back(events::RoomEvent<events::msg::Location>(e));
```